### PR TITLE
commit for ticket 7

### PR DESCRIPTION
**Pull request by rose-phillips**

### CODE REVIEW

File: __tests__/INSERT INTO comments.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO comments
```
Comment (callumu21) [inline __tests__/INSERT INTO comments.sql]: I would make sure that this is .gitignored

File: __tests__/app.test.js [hunk 2]
```diff
@@ -156,3 +124,54 @@ describe("\nGET /api/reviews/:review_id/comments\n", () => {
       });
   });
 });
+describe("\nPOST /api/reviews/:review_id/comments\n", () => {
+  test("get 201 status and add new review comment to database and return new comment", () => {
+    const review_id = 2;
+    const newComment = {
+      review_id: review_id,
```
Comment (callumu21) [inline __tests__/app.test.js]: review_id shouldn't be on the request, just username and body

File: __tests__/app.test.js [hunk 3]
```diff
@@ -156,3 +124,54 @@ describe("\nGET /api/reviews/:review_id/comments\n", () => {
...
+      username: "dav3rid",
+      body: "This is exactly what I was saying to my cat last night.",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body).toMatchObject({
+          body: expect.any(String),
```
Comment (callumu21) [inline __tests__/app.test.js]: Make sure to test against what you know the comment object should look like. You provided the body, so hopefully it comes back the same

File: __tests__/app.test.js [hunk 4]
```diff
@@ -156,3 +124,54 @@ describe("\nGET /api/reviews/:review_id/comments\n", () => {
...
+        });
+      });
+  });
+  test("get 404 'not found' when review_id not found", () => {
+    const review_id = 2000;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
```
Comment (callumu21) [inline __tests__/app.test.js]: GET is a different method to post. You want to check what happens when you POST to the parametric endpoint with a valid but non-existent review_id

File: __tests__/app.test.js [hunk 5]
```diff
@@ -156,3 +124,54 @@ describe("\nGET /api/reviews/:review_id/comments\n", () => {
...
+      username: "Justine",
+      body: "This is exactly what I was saying to my cat last night.",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(404)
+      .then((response) => expect(response.body.msg).toBe("user not found"));
+  });
+});
```
Comment (callumu21) [inline __tests__/app.test.js:236]: Just missing a few more tests here. What happens when username isn't on the request body? What happens if the request body has other, unnecessary keys?

File: models/games-model.js [hunk 6]
```diff
@@ -60,3 +59,24 @@ exports.selectReviewComments = (review_id) => {
...
+      `
+      INSERT INTO comments 
+      (
+        review_id, author, body
+      )
+      VALUES
+      (
+        $1, $2, $3
+      )
+      RETURNING body;
```
Comment (callumu21) [inline models/games-model.js]: We want to return the whole comment here, not just the body.

**Review by callumu21 — CHANGES_REQUESTED** [review]
Just a few changes required before approval. See my comments, and let me know if anything isn't clear

**Comment by Geo-Beard** [comment]
Looks good - all amendments made :) feel free to merge and move on!

**Pull request merged**